packages/unplugin/src/plugin/core.ts

```diff
@@ -68,8 +68,15 @@ export interface PandaPluginOptions extends Partial<PandaPluginHooks>, Pick<Tran
    * Generate a styled-system folder on server start.
    *
    * @default true
+   * @deprecated Use 'codegen' instead
    */
   codeGen?: boolean
+  /**
+   * Generate a styled-system folder on server start.
+   *
+   * @default true
+   */
+  codegen?: boolean
```

Comment:
I also renamed this to have casing consistent with most places "codegen" is used by Panda. Deprecating might be overkill since this option was just added, but I was thinking it's nice to deprecate it at least for a bit since it's not hard.